doc/nrf/libraries/bluetooth_services/mesh/scheduler_cli.rst

```diff
@@ -61,6 +61,74 @@ Persistent storage
 
 None.
 
+Shell commands
+**************
+
+The Bluetooth mesh shell subsystem provides set of commands to interract with the Scheduler Client model instantiated on a device.
+
+To make these commands available, enable :kconfig:`CONFIG_BT_MESH_SHELL` and :kconfig:`CONFIG_BT_MESH_SHELL_SCHEDULER_CLI`.
+
+``mdl_sched instance get-all``
+------------------------------
+
+	Print all instances of the Scheduler Client model on the device.
+
+
+``mdl_sched instance set <elem_idx>``
+-------------------------------------
+
+	Select the Scheduler Client model instantiated on the specified element id. This instance will be used in message sending. If no model instance was selected, the first model instance found on the device will be used by default.
+
+	* ``elem_idx``: Element index where to find the model instance.
+
+
+``mdl_sched get``
+-----------------
+
+	Get the current Schedule Register status.
+
+
+``mdl_sched action-get <idx>``
+------------------------------
+
+	Get the appropriate Scheduler Action status.
+
+	* ``idx``: Index of the Schedule Register entry to get.
+
+
+``mdl_sched action-ctx-set <year> <month> <day> <hour> <minute> <second> <day_of_week> <action> <transition_time> <scene_number>``
+----------------------------------------------------------------------------------------------------------------------------------
+
+	Set the appropriate Scheduler Action and wait for a response.
```

Comment:
```suggestion
	Set the appropriate Scheduler Action. Used in combination with ``mdl_sched action-set`` or ``mdl_sched action-set``.
```
Since this call does not actually send anything I think it could be misleading to write `and wait for a response.`